docs/community/governance.md

```diff
@@ -90,36 +90,42 @@ One of their primary responsibilities is merging pull requests after all require
 
-## Area Experts
+## Working Groups (WGs)
 
-Area Experts are people with knowledge of specific components or technologies in the PowerShell domain. They are responsible for code reviews, issue triage, and providing their expertise to others.
+[Working Groups (WGs)][wg] are collections of contributors with knowledge of specific components or
+technologies in the PowerShell domain.
+They are responsible for issue triage/acceptance, code reviews, and providing their expertise to
+others in issues, PRs, and RFC discussions.
 
 They have [write access](https://help.github.com/articles/permission-levels-for-an-organization-repository/) to the PowerShell repository which gives them the power to:
 
 1. `git push` to all branches *except* `master`.
 1. Merge pull requests to all branches *except* `master` (though this should not be common given that [`master`is the only long-living branch](../git/README.md#understand-branches)).
 1. Assign labels, milestones, and people to [issues](https://guides.github.com/features/issues/).
 
-A list of Area Experts can be found [here][experts].
-
-### Area Expert Responsibilities
+### Working Group Responsibilities
 
-If you are an Area Expert, you are expected to be actively involved in any development, design, or contributions in your area of expertise.
+If you are a member of a Working Group, you are expected to be actively involved in any development, design, or contributions in the focus area of the WG.
+More information on the responsibilities of Working Groups can be found [here][wg],
+while current WG definitions and membership can be found [here][wg-definitions].
 
-If you are an Area Expert:
+If you are a Working Group member:
 
-1. **DO** assign the [correct labels][issue-process]
-1. **DO** assign yourself to issues labeled with your area of expertise
-1. **DO** code reviews for issues where you're assigned or in your areas of expertise.
-1. **DO** reply to new issues and pull requests that are related to your area of expertise
-  (while reviewing PRs, leave your comment even if everything looks good - a simple "Looks good to me" or "LGTM" will suffice, so that we know someone has already taken a look at it).
-1. **DO** make sure contributors are following the [contributor guidelines](../../.github/CONTRIBUTING.md).
-1. **DO** ask people to resend a pull request, if it [doesn't target `master`](../../.github/CONTRIBUTING.md#lifecycle-of-a-pull-request).
+1. **DO** triage and contribute to discussions in issues and PRs that have `WG-*` labels assigned
+1. **DO** regularly communicate with other members of your WG to coordinate decisions about
+   whether issues should move forward
+1. **DO** make decisions on whether or not issues should proceed forward with implementations or RFCs
+1. **DO** assign the [correct labels][issue-process] to issues
+1. **DO** assign yourself to issues labeled with your area of expertise when you plan to implement
```

Comment:
I think we should have a bit stronger language here, people should assign issues to themselves (note that I think only Contributors have this permission or show up in the list) if they are ACTIVELY working on the issue.  We want to avoid people not working on issues because someone else self-assigned (or assigned to someone else) and no one is actually working on it.